src/api/repository/mongo.py

```diff
@@ -0,0 +1,12 @@
+from flask_pymongo import PyMongo
+from typing import List
+from api.repository import validate_input
+
+
+def insert_to_db(mongo: PyMongo, valid_data: List[dict]):
```

Comment:
since it's private method, add '_' prefix so it won't be imported or called by accident by anyone

Reply:
OK